flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/operator/AbstractKeyedCEPPatternOperator.java

```diff
@@ -100,27 +119,21 @@ public void open() throws Exception {
 
 		if (nfaOperatorState == null) {
 			nfaOperatorState = getPartitionedState(
-					new ValueStateDescriptor<NFA<IN>>(
-						NFA_OPERATOR_STATE_NAME,
-						new NFA.Serializer<IN>()));
+				new ValueStateDescriptor<>(NFA_OPERATOR_STATE_NAME, new NFA.Serializer<IN>()));
 		}
 
 		@SuppressWarnings("unchecked,rawtypes")
 		TypeSerializer<StreamRecord<IN>> streamRecordSerializer =
-				(TypeSerializer) new StreamElementSerializer<>(getInputSerializer());
+			(TypeSerializer) new StreamElementSerializer<>(getInputSerializer());
 
 		if (priorityQueueOperatorState == null) {
 			priorityQueueOperatorState = getPartitionedState(
-					new ValueStateDescriptor<>(
-						PRIORIRY_QUEUE_STATE_NAME,
-						new PriorityQueueSerializer<>(
-								streamRecordSerializer,
-								new PriorityQueueStreamRecordFactory<IN>())));
+				new ValueStateDescriptor<>(PRIORITY_QUEUE_STATE_NAME,
+					new PriorityQueueSerializer<>(streamRecordSerializer, new PriorityQueueStreamRecordFactory<IN>())));
```

Comment:
If doing reformatting changes then please try to not change a consistent style. When breaking long parameter lists, imo, every parameter should be on a separate line and indented identically. This is not the case with `PRIORITY_QUEUE_STATE_NAME`.